diff --git a/CMakeLists.txt b/CMakeLists.txt
index 367bfc4..007f732 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -1,56 +1,10 @@
-cmake_minimum_required(VERSION 2.8.3)
+cmake_minimum_required(VERSION 3.5)
 project(realsense_gazebo_description)
 
-# Load catkin and all dependencies required for this package
-find_package(catkin REQUIRED COMPONENTS
-    roscpp
-    gazebo_ros
-    sensor_msgs
-    image_transport
-    camera_info_manager
-    realsense_gazebo_plugin
-    realsense2_description
-)
-find_package(gazebo REQUIRED)
-
-set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} ${GAZEBO_CXX_FLAGS}")
-
-###################################
-## catkin specific configuration ##
-###################################
-## The catkin_package macro generates cmake config files for your package
-## Declare things to be passed to dependent projects
-## INCLUDE_DIRS: uncomment this if your package contains header files
-## LIBRARIES: libraries you create in this project that dependent projects also need
-## CATKIN_DEPENDS: catkin_packages dependent projects also need
-## DEPENDS: system dependencies of this project that dependent projects also need
-catkin_package(
-#  LIBRARIES ${PROJECT_NAME}
-  CATKIN_DEPENDS
-        roscpp
-        gazebo_ros
-        sensor_msgs
-        image_transport
-        camera_info_manager
-        realsense_gazebo_plugin
-        realsense2_description
-)
-
-###########
-## Build ##
-###########
-
-## Specify additional locations of header files
-## Your package locations should be listed before other locations
-include_directories(
-#  include
-  ${catkin_INCLUDE_DIRS}
-)
+find_package(ament_cmake REQUIRED)
 
-install(DIRECTORY urdf
-  DESTINATION ${CATKIN_PACKAGE_SHARE_DESTINATION}
+install(DIRECTORY meshes urdf
+        DESTINATION share/${PROJECT_NAME}
 )
 
-install(DIRECTORY meshes
-  DESTINATION ${CATKIN_PACKAGE_SHARE_DESTINATION}
-)
\ No newline at end of file
+ament_package()
\ No newline at end of file
diff --git a/package.xml b/package.xml
index 17943d2..ded9c8f 100644
--- a/package.xml
+++ b/package.xml
@@ -6,18 +6,10 @@
   <maintainer email="michele.tartari@iit.it">m-tartari</maintainer>
   <author email="michele.tartari@iit.it">m-tartari</author>
   <license>Apache 2.0</license>
-  
-  <buildtool_depend>catkin</buildtool_depend>
-  <depend>roscpp</depend>
-  <depend>gazebo_ros</depend>
-  <depend>sensor_msgs</depend>
-  <depend>image_transport</depend>
-  <depend>camera_info_manager</depend>
-  <depend>realsense_gazebo_plugin</depend>
-  <depend>realsense2_description</depend>
 
-  <!-- The export tag contains other, unspecified, tags -->
+  <buildtool_depend>ament_cmake</buildtool_depend>
+
   <export>
-    <!-- Other tools can request additional information be placed here -->
+    <build_type>ament_cmake</build_type>
   </export>
-</package>
\ No newline at end of file
+</package>
